COPYING

```diff
@@ -1,31 +1,26 @@
-This program is free software: you can redistribute it and/or modify it 
-under the terms of the GNU General Public License as published by the 
-Free Software Foundation, either version 3 of the License, or (at your 
-option) any later version, along with the Special Exception for use of 
-the program along with SSL, set forth below. This program is distributed 
-in the hope that it will be useful, but WITHOUT ANY WARRANTY; 
-without even the implied warranty of MERCHANTABILITY or FITNESS FOR 
-A PARTICULAR PURPOSE.  See the GNU General Public License for more 
-details. You should have received a copy of the GNU General Public 
-License along with this program.  If not, see 
-<http://www.gnu.org/licenses/>.
-.
-The author can be reached by email at clearwater@metaswitch.com or by 
-post at Metaswitch Networks Ltd, 100 Church St, Enfield EN2 6BQ, UK
+Source: http://projectclearwater.org
+Copyright: Metaswitch Networks 2017
+.
+The author can be reached by email at clearwater@metaswitch.com or by post at
+Metaswitch Networks Ltd, 100 Church St, Enfield EN2 6BQ, UK
+.
```

Comment:
Because it's required.   In fact re-reading the spec, the continuation lines all need to be indented with a space, and we do need a Files field.  I'll fix this.